docs/reference/search/run-a-search.asciidoc

```diff
@@ -0,0 +1,284 @@
+// TESTRESPONSE[s/"took": 2/"took": "$body.took"/]
+====
+
+[discrete]
+[[run-request-body-search]]
+=== Run a request body search
+
+You can use the search API's <<search-api-request-body-query,`query` request
+body parameter>> to provide a query as a JSON object, written in
+<<query-dsl,Query DSL>>.
+
+.*Example*
+[%collapsible]
+====
+The following request body search uses the <<query-dsl-match-query,`match`>>
+query to match documents with a `message` value of `login successful`. Note the
+`match` query is specified as a JSON object in the `query` parameter.
+
+[source,console]
+----
+GET /user_logs_000001/_search
+{
+  "query": {
+    "match": {
+      "message": "login successful"
+    }
+  }
+}
+----
+// TEST[continued]
+
+The API returns the following response.
+
+The `hits.hits` property contains matching documents. By default, the response
+sorts these matching documents by `_score`, a <<relevance-scores,relevance
+score>> that measures how well each document matches the query.
+
+[source,console-result]
+----
+{
+  "took": 1,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 3,
+      "relation": "eq"
+    },
+    "max_score": 0.9983525,
+    "hits": [
+      {
+        "_index": "user_logs_000001",
+        "_id": "2",
+        "_score": 0.9983525,
+        "_source": {
+          "@timestamp": "2020-12-07T11:06:07.000Z",
+          "user": {
+            "id": "8a4f500d"
+          },
+          "message": "Login successful"
+        }
+      },
+      {
+        "_index": "user_logs_000001",
+        "_id": "3",
+        "_score": 0.49917626,
+        "_source": {
+          "@timestamp": "2020-12-07T11:07:08.000Z",
+          "user": {
+            "id": "l7gk7f82"
+          },
+          "message": "Logout successful"
+        }
+      },
+      {
+        "_index": "user_logs_000001",
+        "_id": "1",
+        "_score": 0.42081726,
+        "_source": {
+          "@timestamp": "2020-12-06T11:04:05.000Z",
+          "user": {
+            "id": "vlb44hny"
+          },
+          "message": "Login attempt failed"
+        }
+      }
+    ]
+  }
+}
+----
+// TESTRESPONSE[s/"took": 1/"took": "$body.took"/]
+====
+
+[discrete]
+[[search-multiple-indices]]
+=== Search multiple indices
+
+To search multiple indices, add them as comma-separated values in the search API
+request path.
+
+.*Example*
+[%collapsible]
+====
+The following request searches the `user_logs_000001` and `user_logs_000002`
+indices.
+
+[source,console]
+----
+GET /user_logs_000001,user_logs_000002/_search
+{
+  "query": {
+    "match": {
+      "message": "login successful"
+    }
+  }
+}
+----
+// TEST[continued]
+// TEST[s/^/PUT user_logs_000002\n/]
+====
+
+You can also search multiple indices using a glob or wildcard expression.
+
+.*Example*
+[%collapsible]
+====
+The following request uses the glob `user_logs*` in place of the index name.
```

Comment:
This isn't specifically related to this reorg, but I've been wondering about the use of "glob" vs "index pattern". Given the prevalence of index pattern in other contexts, it feels like the glob terminology is potentially confusing. Glob isn't wrong in this context, but I'd be inclined to use index pattern, anyway.

Reply:
Solid point. Removed `glob` and used `index pattern` throughout. Thanks @debadair!